src/index.ts

```diff
@@ -0,0 +1,19 @@
+// @flow
```

Comment:
you can actually get rid of this file now if you set `[cli-engine].commands` to `./lib/commands`. Bonus is that it will soon be much faster in linked too